### ISSUE TITLE
Asteroid MoMMI related requests.

### ISSUE THREAD

**Issue by AssBlastProductions** [Zaers/-tg-station#63]
Probably most of the fun in being a MoMMI in any way is getting to build crazy bullshit and improve on the station you're assigned to. This becomes a bit of a problem with asteroid MoMMI's specifically since being locked to z-level 5 means no access to certain essential basics that are needed to expand and improve and build new rooms in general such as for example atmospherics. Specifically power control modules, powercells and air alarm electronics and to a lesser extent other not quite essential things such as security cameras. 

To improve on the situation and allow asteroid MoMMI's to properly expand the mining station without having to pray for somebody to get borged and drop off materials I'd like to request that either a vendor similar to the one in engineering that contains the electronics for fire alarms, air alarms and so on be added on one of the outposts on the asteroid or alternatively that an autolathe is added somewhere so that basic building materials such as electronics and cameras could be printed out for the purpose of expanding and improving the asteroid station. 

I don't think this would cause any balancing issues regarding other non MoMMI personnel on the asteroid since miners already have access to an autolathe and scientists can flat out construct one. Additionally regardless of whether an autolathe is added to the asteroid or not I'd like to request that autolathes be made capable of printing out the most basic default powercells since it seems like something as common and mundane as a basic powercell should be possible to produce somehow outside of research or ordering refill crates.

In short please add an autolathe or vendor containing electronics to the asteroid so MoMMI's can actually improve it. Also if possible remove the wooden barricade that blocks the smelter on the asteroid because fuck that thing and having to dismantle windows.

**Comment by octareenroon91** [comment]
Looking at the datum for powercells, it looks like the cell is halfway there already.

https://github.com/Zaers/-tg-station/blob/master/code/modules/research/designs/power_designs.dm#L10
